version.txt

```diff
@@ -0,0 +1 @@
+0.0.1-rc.1
```

Comment:
Can we bump up to version 1.0.0?

Reply:
We can do whatever we want to the version because I'm not creating tags with that versioning scheme. I plan on circling back and implementing [calendar versioning](https://calver.org/) because semantic versioning doesn't make sense with everything this repository contains.

Reply:
Please see #874.